lib/src/DatePicker/DatePicker.jsx

```diff
@@ -25,6 +25,8 @@ export class DatePicker extends PureComponent {
     utils: PropTypes.object.isRequired,
     shouldDisableDate: PropTypes.func,
     allowKeyboardControl: PropTypes.bool,
+    multi: PropTypes.bool,
```

Comment:
It seems as though these types of selections are wrappers that replace the `renderDay` implementation. Might it make sense to implement a separate `MultiDatePickerWrapper` and `RangeDatePickerWrapper`?

Reply:
I went that direction with my first attempt. I got it somewhat working but there where issues and ended up hacky all around. Rolling the features into the core seamed to be a better solution. Honestly it required a lot less modification than I expected.